.github/ansible/redeploy-derper.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Update derper node
+  hosts: derper
+  become: yes
+
+  tasks:
+  - name: Make sure derper is stopped
+    ansible.builtin.systemd:
+      state: stopped
+      enabled: yes
+      name: derper
+  - name: Fetch derper binary
+    get_url:
+      url: https://vorc.s3.us-east-2.amazonaws.com/derper-linux-amd64-{{ derper_version }}
+      mode: '0755'
+      force: yes
+      dest: /usr/local/bin/derper
+  - name: Allow ports
+    shell:
+      cmd: sudo setcap CAP_NET_BIND_SERVICE=+eip /usr/local/bin/derper
+  - name: Make sure derper is started
+    ansible.builtin.systemd:
+      state: restarted
```

Comment:
it seems to me that this is the only systemd interaction needed.  The stopping earlier can only result in a stopped service if the download fails I'd expect (I'm guessing at how ansible works here).  But there should be nothing wrong with downloading the binary while the service is still running.

Reply:
Let's try 🤷